Tests/TestCodeGenConfigurations/SPMInXcodeProject/apollo-codegen-config.json

```diff
@@ -57,7 +57,7 @@
       "path" : "./AnimalKingdomAPI",
       "moduleType" : {
         "swiftPackageManager" : {
-
+          
```

Comment:
Should this one be changed to the new `swiftPackage` too?

Reply:
Yea might as well, not actually required since this gets decoded into the new one but worth going ahead and switching it.

Reply:
Actually looks like this one may require the change because of the need for the local path causing the ci failure